openmetadata-ui/src/test/java/org/openmetadata/catalog/selenium/pagesWithoutData/myData/MyDataPageTest.java

```diff
@@ -0,0 +1,127 @@
+package org.openmetadata.catalog.selenium.pagesWithoutData.myData;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Order;
+import org.junit.jupiter.api.Test;
+import org.openmetadata.catalog.selenium.events.Events;
+import org.openmetadata.catalog.selenium.properties.Property;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.chrome.ChromeOptions;
+import org.openqa.selenium.interactions.Actions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+import java.time.Duration;
+import java.util.Objects;
+
+public class MyDataPageTest {
+
+  static WebDriver webDriver;
+  static String url = Property.getInstance().getURL();
+  static Actions actions;
+  static WebDriverWait wait;
+  static String table = "dim_product_variant";
+  Integer waitTime = Property.getInstance().getSleepTime();
+
+  @BeforeEach
+  public void openMetadataWindow() {
+    System.setProperty("webdriver.chrome.driver", "src/test/resources/drivers/macM1/chromedriver");
+    ChromeOptions options = new ChromeOptions();
+    options.addArguments("--headless");
+    options.addArguments("--window-size=1280,800");
+    webDriver = new ChromeDriver(options);
+    actions = new Actions(webDriver);
+    wait = new WebDriverWait(webDriver, Duration.ofSeconds(30));
+    webDriver.manage().window().maximize();
+    webDriver.get(url);
+  }
+
+  @Test
+  @Order(1)
+  public void checkWhatsNew() {
+    Events.click(webDriver, By.xpath("//ul[@class='slick-dots testid-dots-button']//li[2]")); // What's new page 2
+    Events.click(webDriver, By.cssSelector("[data-testid='WhatsNewModalChangeLogs']")); // Change Logs
+    Events.click(webDriver, By.cssSelector("[data-testid='closeWhatsNew']")); // Close What's new
+  }
+
+  @Test
+  public void checkOverview() throws Exception {
+    checkWhatsNew();
+    String tablesCount = webDriver.findElement(By.xpath("//div[@data-testid='tables-summary']//span[@data-testid='filter-count']")).getAttribute("innerHTML");
+    if (!Objects.equals(tablesCount, "0")) {
+      throw new Exception("Count should be 0");
```

Comment:
Why is [Assert](http://junit.sourceforge.net/javadoc/org/junit/Assert.html) not used in selenium tests?

Reply:
I've added to few, will add here as well as to all other test cases.